h2d/Scene.hx

```diff
@@ -289,13 +264,48 @@ class Scene extends Layers implements h3d.IDrawable implements hxd.SceneEvents.I
 			}
 			if( !visible ) continue;

```

Comment:
I would like to keep width/height testing before visibility testing, in case there are many Interactive within deep hierarchies.

Reply:
And for case of shape interactive? Should it check for visibility before or after shape collision check?

Reply:
I guess we should be uniform here and check after in all cases